config/__init__.py

```diff
@@ -122,9 +122,11 @@ def update_status_config(milestones_content):
 if not PRODUCTION:
     DEBUG = True
 
-# localhost runs on HTTP, use secure flag on session cookie otherwise.
+# localhost runs on HTTP
+# use secure and samesite flags on session cookie otherwise
 if not LOCALHOST:
     SESSION_COOKIE_SECURE = True
+    SESSION_COOKIE_SAMESITE = 'Lax'
```

Comment:
Does `Strict` break things?

Reply:
The flask docs said `Lax` was recommended because
> `Strict` prevents sending cookies with all external requests, including following regular links.

That sounded potentially "breaky" to me. We set cookies to keep track of login state, don't we?

Reply:
I guess it's worth testing... the cookies we use to track login state only make sense on webcompat.com, them not being sent to GitHub shouldn't matter.

The only open question is if logging in the initial time will still work, but I suspect it will -- because it sends back auth token junk in the URL. But should be easy to test by removing the local oauth app, and re-authenticating with `Strict` set.

In general, I think it's good to be as secure as possible, so it's worth testing.

Reply:
OK, so I don't really understand what's going on, but it seems like even with `Lax`, I'm having problems logging into GH, but it's GH giving me the 404. Is that because we're not managing to pass it something? Or is it just me? 😭  I'm so confused by which file powers what in testing vs production/staging (comment: #2518 ), so I'm not sure I disabled/enabled everything I needed to. Advice appreciated.

<img width="691" alt="screenshot 2018-07-17 19 33 20" src="https://user-images.githubusercontent.com/20972610/42835134-772ac540-89f8-11e8-95b0-c3e7faa70c06.png">

<img width="918" alt="screenshot 2018-07-17 19 33 41" src="https://user-images.githubusercontent.com/20972610/42835135-774ca1ba-89f8-11e8-81a2-618650916980.png">